src/assets/locales/en.json

```diff
@@ -1274,7 +1275,8 @@
       "selfAlliesAffectRange": "%game.affect% self and all allies within %game.action.range% {0}",
       "selfAlliesRange": "%game.target% self and all allies within %game.action.range% {0}",
       "selfOrAlly": "Self or ally",
-      "targets": "All targets"
+      "targets": "All targets",
+      "targetAttackAbility": "%game.target% 1 target of the attack<br>ability at any %game.action.range%, %game.action.pierce:1%"
```

Comment:
Please remove this, seems specialized for a specific item, the `specialTarget` is really for generic target, with the `%game.action.pierce:1%` this is not a target only ability.

Another small hint: `%game.action.pierce:1%` will not work, just use `%game.action.pierce% 1`